src/app/(accounts)/layout.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+
+import { Route } from "@/routers/types";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import React from "react";
+import { FC } from "react";
+
+export interface CommonLayoutProps {
+  children?: React.ReactNode;
+}
+
+const pages: {
```

Comment:
u can create a navigation.constants and place this object; 
Because this makes maintenance easy everywhere